src/include/txn.h

```diff
@@ -19,8 +19,9 @@
  * the session get rollback reason API call. Users of the API could have a dependency on the format
  * of these messages so changing them must be done with care.
  */
-#define WT_TXN_ROLLBACK_REASON_CACHE "oldest pinned transaction ID rolled back for eviction"
 #define WT_TXN_ROLLBACK_REASON_CONFLICT "conflict between concurrent operations"
+#define WT_TXN_ROLLBACK_REASON_OLDEST_FOR_EVICTION \
```

Comment:
I don't think we should rename this macro. This was done as part of another change: https://github.com/wiredtiger/wiredtiger/commit/de7db74e12cdedd6d026e7674c466d95e47024bd

We should keep the same name as before, and hence no reordering. This naming also matches the corresponding entry in MongoDB code:
https://github.com/mongodb/mongo/blob/v6.0/src/mongo/db/storage/wiredtiger/wiredtiger_util.cpp#L64

Let's keep the name.

Reply:
addressed [f789e91](https://github.com/wiredtiger/wiredtiger/pull/8665/commits/f789e9155171960af9babea55760494256cc3ff7)